modules/localgov_netcall_ai_widget/js/ai-widget-init.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * @file JS file for the Netcall AI Widget.
+ *
+ * This script is responsible for loading the Netcall AI widget onto the page.
+ * It gets placed in the <head> tag.
+ */
+
+(function netcallAiWidgetInitScript(Drupal, drupalSettings) {
+  Drupal.behaviors.netcallAiWidgetInit = {
+    attach(context) {
+      const workspaceId = drupalSettings.localgovNetcall.aiWigdetWorkspaceId;
+      let partitionId = drupalSettings.localgovNetcall.aiWigdetPartitionId;
+      const currentPath = window.location.pathname;
+      const partitionOverrides = [];
+      const partitionOverridesFromConfig =
+        drupalSettings.localgovNetcall.aiWigdetPartitionOverrides;
+
+      // Check if there's any overrides in the config.
+      // If so, add each to the partitionOverrides array.
+      if (partitionOverridesFromConfig) {
+        partitionOverridesFromConfig
+          .split('\r\n')
```

Comment:
I hope `\r\n` works for everybody :)

Reply:
If we get any complaints, we can figure it out then.